main.go

```diff
@@ -406,7 +424,7 @@ func main() {
 				}
 			}
 
-			oidcHandler, oidcTenantMiddlewares, warnings := authentication.NewOIDC(logger, oidcs)
+			oidcHandler, oidcTenantMiddlewares, warnings := authentication.NewOIDC(logger, "/oidc/{tenant}", oidcs)
```

Comment:
How is this related to tracing? I guess we also updated dependencies. Could we do that in a separate PR?

Reply:
I wanted to set proper route tags but we have a lot of nested routers. I am passing the route prefix from here because I didn't want to hardcode it.

The dependency bump was not intentional, I guess a few of them got updated when I added the OTel dependencies. Anyways, I'll try to revert that.